packages/ui/src/TextArea.tsx

```diff
@@ -10,6 +10,6 @@ export class TextArea extends React.Component<TextAreaProps, {}> {
   }
 
   render() {
-    return <TextField {...this.props} height={100} multiline rows={4} />;
```

Comment:
Nice, this is great! Can you make height default to 100?

Reply:
Is there a way to do that inline? I'm seeing it's possible to do use something like TextArea.defaultProps

Reply:
Not that I know of. Probably something like `const {height, ...props} = this.props` then `height={height ?? 100}`